source/test/regex_registry.test.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { getRegexRegistry, RegexTestCase } from "./regex_registry";
+import { getRegexRegistry, RegexTestCase } from './regex_registry';
 
 const testCases: RegexTestCase[] = getRegexRegistry().getAllCases();
```

Comment:
Do I understand correctly, the regexes in here exist in some of our production files (documents), and have been copied here for testing?
I understand that you don't want to make changes, I'd appreciate 2-3 sentences of explanation in the file.

Reply:
Yes. With this set of tests, a regex that exists in our ssm docs must also exist in these tests. Not all of them have test cases because there were too many to add at the time, and there is significant benefit of preventing us from inventing new regexes to test the same values.
The rest of these tests test a number of aspects of our documents that are expected to be uniform (like taking a role as input).